packages/nx/src/devkit-exports.ts

```diff
@@ -196,7 +196,12 @@ export type { JsonParseOptions, JsonSerializeOptions } from './utils/json';
 /**
  * @category Utils
  */
-export { readJsonFile, writeJsonFile } from './utils/fileutils';
+export {
```

Comment:
Please revert this bit, this is adding to our public API I believe, let's keep the changes internal